src/app/lists/CommentBox.tsx

```diff
@@ -92,16 +101,17 @@ const CommentBox: React.FC<CommentProps> = ({ originalComment }) => {
             window.location.reload();
         }
     };
+
     return (
         <Wrapper>
             <HeaderAndButtonContainer>
-                <h2>Comments</h2>
```

Comment:
Ooh i think this might be causing the problem? Maybe the test sees an h3 after seeing an h1 without seeing an h2?